From b3873ebfff8c526cbcd8621bbb405eb6673da00d Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Fri, 7 Feb 2020 13:11:31 +0200
Subject: [PATCH 1/2] USSD: Fix alert dialog theme

  * from
  https://github.com/Havoc-OS/android_packages_services_Telephony/commit/5044eaaade65aca046a6254fa83c3d0f7e25aa40

	(partially solution cuz we have problem now with latest MMI result)

	modified:   src/com/android/phone/PhoneUtils.java
---
 src/com/android/phone/PhoneUtils.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/com/android/phone/PhoneUtils.java b/src/com/android/phone/PhoneUtils.java
index 20c8276..986dcef 100644
--- a/src/com/android/phone/PhoneUtils.java
+++ b/src/com/android/phone/PhoneUtils.java
@@ -88,7 +88,7 @@ public class PhoneUtils {
      * because these dialogs are displayed using the application context, which does not resolve
      * the dialog theme correctly.
      */
-    private static final int THEME = com.android.internal.R.style.Theme_DeviceDefault_Dialog_Alert;
+    private static final int THEME = com.android.internal.R.style.Theme_DeviceDefault_Light_Dialog_Alert;
 
     /** USSD information used to aggregate all USSD messages */
     private static AlertDialog sUssdDialog = null;
-- 
2.11.0

